test/e2e/framework/framework.go

```diff
@@ -70,6 +71,8 @@ func (f *Framework) CreateGameServerAndWaitUntilReady(ns string, gs *v1alpha1.Ga
 		return nil, fmt.Errorf("creating %v GameServer instances failed (%v): %v", gs.Spec, gs.Name, err)
 	}
 
+	logrus.WithField("name", newGs.ObjectMeta.Name).Info("GameServer created, waiting for Ready")
```

Comment:
@Kuqd I added some logging here, to make it easier to track down what the name of the GameServers are, if things go wrong.

Reply:
make sense !